app/admin/dashboard.rb

```diff
@@ -0,0 +1,44 @@
+ActiveAdmin.register_page 'Dashboard' do
+  menu priority: 1, label: proc { I18n.t('active_admin.dashboard') }
+
+  content title: proc { I18n.t('active_admin.dashboard') } do
+    div class: 'blank_slate_container', id: 'dashboard_default_message' do
+      span class: 'blank_slate' do
+        span I18n.t('active_admin.dashboard_welcome.welcome')
+        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+      end
+    end
```

Comment:
Just as a suggestion: you can remove this block so it doesn't show the blank welcome message in your dashboard page.

Reply:
Thanks, I was wondering if that part was necessary jaja